repos/spack_repo/builtin/packages/cubelib/package.py

```diff
@@ -33,7 +33,20 @@ def url_for_version(self, version):
 
         return url.format(version.up_to(2), version)
 
+    def clean_compiler(self, compiler):
+        renames = {
+            "cce": "cray",
+            "rocmcc": "amdclang",
+            "intel-oneapi-compilers": "oneapi",
+            "llvm": "clang",
+        }
+        if compiler in renames:
+            return renames[compiler]
+        return compiler
+
```

Comment:
I am also unsure what to do with this as a multi-project convenience function; when a Spack maintainer has a look I would happily take suggestions to avoid the duplication.